src/pretix/plugins/stripe/payment.py

```diff
@@ -747,6 +749,13 @@ class StripeCC(StripeMethod):
     public_name = _('Credit card')
     method = 'cc'
 
+    @property
+    def walletqueries(self):
+        # ToDo: Check against Stripe API, if ApplePay and GooglePay are even activated/available
```

Comment:
Should we offer a checkbox in pretix payment settings in the meantime, like we do for other payment methods as well?

Reply:
So just a "check for Google Pay/Apple Pay availability"-checkbox?

Reply:
yeah